csuntechlab/iSTART
# STR-43 -- DO: Fix Docker-Compose so that Docker works the way it was intended + Laravel Echo

# Description
Adjusted docker-compose.yml and Dockerfile. Dockerfile installs redis and laravel echo server.
https://blog.mikesir87.io/2018/07/leveraging-multi-stage-builds-single-dockerfile-dev-prod/

I will slim down Dockerfile in next commit.
# Testing
1. Checkout the STR-43 branch.
2.  Make sure Dockerfile+docker-compose.yml is inside iSTART directory
3. Run `docker-compose up -d`
4. Give redis and echo a few minutes to serve up.
5. Run the command `docker logs iSTART_echo` to see if laravel echo is "Server ready!"
# Installation
1. Have Dockerfile+docker-compose.yml file.
